### ISSUE TITLE
build fails with :processDebugResourcesC:\Devel\timer\platforms\android\res\xml\config.xml:45 : AAPT: Error parsing XML: unbound prefix  FAILED

### ISSUE THREAD

**Issue by samodadela** [mircoc/cordova-plugin-settings-hook#3]
Hi,

After using this plugin I got the following error:
:processDebugResourcesC:\Devel\timer\platforms\android\res\xml\config.xml:45 : AAPT: Error parsing XML: unbound prefix
 FAILED

What I found is that the "ionic build" (which uses "cordova build" underneath) command copies **everything** it finds in the <platform name="android"> section in the 'main' config.xml (the one in the root of your project) to platforms\android\res\xml\config.xml. I had this: 
```
 <platform name="android">

    <config-file target="AndroidManifest.xml" parent="/*">
        <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION"/>
        <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
        <uses-permission android:name="android.permission.ACCESS_LOCATION_EXTRA_COMMANDS"/>
    </config-file>

```

So this got copied to platforms\android\res\xml\config.xml causing an error because it did not know about android:name. The workaround is to add this to the top of your 'main' config file:
`xmlns:android="http://schemas.android.com/apk/res/android"`

Why did I say workaround? Because I think (correct me if I'm wrong) that the <config-file> tag belongs only to cordova-plugin-settings-hook and it should not be written/copied into the platform config.xml (platforms\android\res\xml\config.xml). The workaround works because it makes the platform config.xml parsable.

A better way would probably be to put cordova-plugin-settings-hook's settings under some other tag which won't end up being copied to the platform's config.xml. For example:
```
<platform-cordova-settings-hook name="android">
   <config-file target="AndroidManifest.xml" parent="/*">
```

This issue is just FYI - no need to change implement - probably not worth the effort since the workaround works nicely. Maybe note it in the readme?

Thanks to AshConnel for the solution. [Cordova/Ionic CLI hooks for modifying AndroidManifest.xml...](https://forum.ionicframework.com/t/cordova-ionic-cli-hooks-for-modifying-androidmanifest-xml-or-ios-plist-file/12293/19)

**Comment by mircoc** [comment]
I think that the best solution is to add the namespace definition in the root of config.xml as you found out. I know that is a workaround, but i think that anyway all that is written on that file get copied to res/ folder.

**Comment by samodadela** [comment]
Yes, you are right - everything gets copied, not only the platform=android section.
Thanks for updating the readme.
Close it.
